src/api/api.py

```diff
@@ -98,17 +185,33 @@ def update_record(username: str, num_new_comments: int) -> bool:
         return False
 
 
-def create_record(username: str, num_runs: int, num_comments: int) -> bool:
+def create_record(
+    username: str,
+    num_runs: int,
+    num_comments: int,
+    is_comm_blocked: bool,
+    is_like_blocked: bool,
+) -> bool:
 
-    stats_table = Table(airtable_key, base_id, "Stats")
+    stats_table = Table(airtable_key, base_id, STATS_TABLE)
 
     try:
         record = {
-            "Username": username,
-            "NumRuns": num_runs,
-            "NumComments": num_comments,
+            USERNAME_COL: username,
+            NUM_RUNS_COL: num_runs,
+            NUM_COMM_COL: num_comments,
         }
 
+        if is_comm_blocked:
+            record[NUM_COMM_BLOCKED_COL] = int(is_comm_blocked)
+        if is_like_blocked:
+            record[NUM_LIKE_BLOCKED_COL] = int(is_like_blocked)
+
+        if is_comm_blocked:
+            record[NUM_COMM_BLOCKED_COL] = int(is_comm_blocked)
+        if is_like_blocked:
+            record[NUM_LIKE_BLOCKED_COL] = int(is_like_blocked)
+
```

Comment:
Are these dupes? Also is there a reason we can't just have these fields in the json you create above?

```suggestion
        if is_comm_blocked:
            record[NUM_COMM_BLOCKED_COL] = int(is_comm_blocked)
        if is_like_blocked:
            record[NUM_LIKE_BLOCKED_COL] = int(is_like_blocked)
```

Reply:
The reason is because if we always put it into that json (even when the user didnt get comment blocked or liked blocked), this would update the column for LastTimeCommentBlocked or LastTimeLikeBlocked. We only want those columns to be updated when someone is_comm_blocked or someone is_like_blocked

Reply:
Ah I see, so those fields are automatically updated when we write to them in Airtable?